<div class="chatlog__message-group">
  <div id="chatlog__message-container-535717344085934081" class="chatlog__message-container" data-message-id="535717344085934081">
    <div class="chatlog__message">
      <div class="chatlog__message-aside"><img class="chatlog__avatar" src="https://cdn.discordapp.com/avatars/189711109966659584/95280765d1be73ce985dce15ea3585f6.png?size=512" alt="Avatar" loading="lazy"></div>
      <div class="chatlog__message-primary">
        <div class="chatlog__header"><span class="chatlog__author" style="color:rgb(237,171,161)" title="omochimetaru#1122" data-user-id="189711109966659584">omochimetaru</span> <a href="/channels/375206337937801216?category=main&amp;channel=swift-zatsu&amp;message_id=535717553050484736"><span class="chatlog__timestamp"></span></a><a href="#chatlog__message-container-535717344085934081">18-Jan-19 07:09 AM</a></div>
        <div class="chatlog__content chatlog__markdown"><span class="chatlog__markdown-preserve">XCTestCaseでテストメソッドからSwiftの例外を投げると、</span></div>
      </div>
    </div>
  </div>
  <div id="chatlog__message-container-535717437941743618" class="chatlog__message-container" data-message-id="535717437941743618">
    <div class="chatlog__message">
      <div class="chatlog__message-aside">
        <div class="chatlog__short-timestamp" title="18-Jan-19 07:09 AM">07:09</div>
      </div>
      <div class="chatlog__message-primary">
        <div class="chatlog__content chatlog__markdown"><span class="chatlog__markdown-preserve">XCTestのランタイム側でSwiftのエラーがNSErrorに変換されて？から、コンソールにprintされるらしくて</span></div>
      </div>
    </div>
  </div>
  <div id="chatlog__message-container-535717527893049344" class="chatlog__message-container" data-message-id="535717527893049344">
    <div class="chatlog__message">
      <div class="chatlog__message-aside">
        <div class="chatlog__short-timestamp" title="18-Jan-19 07:10 AM">07:10</div>
      </div>
      <div class="chatlog__message-primary">
        <div class="chatlog__content chatlog__markdown"><span class="chatlog__markdown-preserve">たとえもとのエラーがCustomStringConvertibleだったとしても、それが表示されないんですけど</span></div>
      </div>
    </div>
  </div>
  <div id="chatlog__message-container-535717553050484736" class="chatlog__message-container" data-message-id="535717553050484736">
    <div class="chatlog__message">
      <div class="chatlog__message-aside">
        <div class="chatlog__short-timestamp" title="18-Jan-19 07:10 AM">07:10</div>
      </div>
      <div class="chatlog__message-primary">
        <div class="chatlog__content chatlog__markdown"><span class="chatlog__markdown-preserve">みなさんどうしてます？</span></div>
      </div>
    </div>
  </div>
</div>